src/utils/generateArtifacts.js

```diff
@@ -32,6 +32,7 @@ export const generateArtifacts = (artifacts, tab, originalContent) => {
         if (artifact.data.length > 0) {
           generatedArtifacts = artifact.data.map(generatedArtifact => {
             let item = { ...generatedArtifact }
+            item.kind = !item.kind ? 'artifact' : item.kind
```

Comment:
Could you simplify this to 
item.kind = item.kind ?? 'artifact'

Use variable instead of "artifact"